submissions/Yasya23/tiny-js-oop/index.js

```diff
@@ -0,0 +1,61 @@
+class Inhabitants {
+  constructor(species, name, gender) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+  }
+
+  showProperties() {
+    return `${this.species}; ${this.name}; ${this.gender}`;
```

Comment:
This and similar fragments below:
Benefit from `Array` methods. This will make code DRYer.

Reply:
Corrected for an array.